rewards/df-max-out-yield.md

```diff
@@ -4,11 +4,11 @@ description: >-
   Ocean Protocol's Data Farming dApp! 🥋
 ---
 
-# Max Out Volume DF (Purple Belt)
```

Comment:
The same language and titles in the docs are being applied to the How-Tos. As a reader, it becomes really hard to understand which readme is giving me theory (system design) vs. application (how-to). This is bloating the readme/docs and making it really hard to maintain/reduce the complexity of it.

I recommend **the docs `/rewards/` should be focused more on the mechanisms/systems** (and renamed to `/data-farming/`) and focused on describing the why Vs. the How-To docs `/user-guides/` that give step-by-step instructions and examples so you can follow along.

This should make it easier to manage the docs, reduce complexity/duplication, and provide a structure that is intuitive to readers.

Example: `data-farming-101.md` (which I focused on completion) rather than deleting it (reducing) so I can make sure the documentation is whole. However, there is a ton of redundancy, in this case against `rewards/df-basic.md`

**The next steps here is to finish completion, and then focus on reduction/coalescing information**